hyp3_autorift/process.py

```diff
@@ -87,35 +86,57 @@ def get_lc2_path(metadata):
     return band['href'].replace('https://landsatlook.usgs.gov/data/', f'/vsis3/{LANDSAT_BUCKET}/')
 
 
+def get_s2_path(manifest_text: str, scene_name: str) -> str:
+    root = ET.fromstring(manifest_text)
+    elements = root.findall(".//fileLocation[@locatorType='URL'][@href]")
+    hrefs = [element.attrib['href'] for element in elements if
+             element.attrib['href'].endswith('_B08.jp2') and '/IMG_DATA/' in element.attrib['href']]
+    if len(hrefs) == 1:
+        # post-2016-12-06 scene; only one tile
+        file_path = hrefs[0]
+    else:
+        # pre-2016-12-06 scene; choose the requested tile
+        tile_token = scene_name.split('_')[5]
+        file_path = [href for href in hrefs if href.endswith(f'_{tile_token}_B08.jp2')][0]
+    return f'/vsicurl/{S2_GRANULE_DIR}/{file_path}'
+
+
+def get_s2_bbox(path: str):
```

Comment:
This isn't technically tied to Sentinel-2, so we could use a more generic name like `get_bbox_of_raster()`, even if we're only using it for Sentinel-2 at the moment. Thoughts?

Reply:
That sounds good! I'll make that change